infra/resources/_modules/monitoring/action_group.tf

```
@@ -0,0 +1,11 @@
+resource "azurerm_monitor_action_group" "io_com_error_action_group" {
+  name                = "iocom-error-ag"
```

We should follow [the naming convention](https://pagopa.atlassian.net/wiki/spaces/DevEx/pages/1032880258/RFC-DX-005+Naming+convention+nuove+risorse+Azure)

I omitted `location` because it default to `global`.

```suggestion
  name                = "io-p-com-error-ag-01"
```